Makefile

```diff
@@ -44,6 +44,12 @@ provision:
 ssh:
 	vagrant ssh mon0
 
+gofmt-host:
+	@build/scripts/gofmt.sh
+
+gofmt:
```

Comment:
we need to be consistent building and checking the source, so we do this inside the VM